summoner-cli/src/Summoner/Config.hs

```diff
@@ -59,12 +61,13 @@ data ConfigP (p :: Phase) = Config
     , cBench        :: !Decision
     , cPrelude      :: !(Last CustomPrelude)
     , cExtensions   :: ![Text]
-    , cWarnings     :: ![Text]
-    , cGhcOptions   :: ![Text]  -- ^ GHC options to add every stanza
+    , cWarnings     :: ![Text]  -- ^ DEPRECATED: TODO: remove in 1.4.
```

Comment:
Created #374 to track this

Reply:
Thanks, that's very helpful! It's indeed a good idea to create issues to have a smoother deprecation period. Nice!